core/templates/components/state-editor/state-skill-editor/state-skill-editor.directive.ts

```diff
@@ -53,16 +53,23 @@ angular.module('oppia').directive('stateSkillEditor', [
           var _init = function() {
             TopicsAndSkillsDashboardBackendApiService.fetchDashboardDataAsync()
               .then(function(response) {
-                // CategorizedSkills: a dict of categorized ShortSkillSummary.
-                // UntriagedSkillSummaries:  array of Untriaged SkillSummary.
+                /**
+                 * CategorizedSkills: dict.
+                 *   {[topicName: string]:
+                 *      {uncategorized: ShortSkillSummary[];
+                 *      [subtopicName: string]: ShortSkillSummary[];
+                 *      };
+                 *   }
+                 * UntriagedSkillSummaries: list(SkillSummary). A list of all
```

Comment:
This should be a standard line comment on top of the untriagedSkillSummaries line. You can write it as prose, e.g.

"This is a list of SkillSummary objects corresponding to all untriaged skills on the topic-and-skill-dashboard page."

Reply:
Done. PTAL!